python/python/lance/dataset.py

```diff
@@ -838,7 +838,7 @@ def take(
             [self._ds.take(indices, columns, columns_with_transform)]
         )
 
-    def _take_rows(
+    def take_rows(
```

Comment:
@westonpace Is there any background we track this method as a private method?

Reply:
Basically, we've considered rowids to confusing for users right now, as they change when rows are updated / compacted. I think we're open to exposing this because eventually we'd like to make them stable primary keys.